TicketsDemo/Carriages.csv

```diff
@@ -0,0 +1,7 @@
+Id;Type;DefaultPrice;TrainId;Number
```

Comment:
please, ensure all data resources are not present in git repository and added to .gitignore